packages/console/src/pages/GetStarted/hook.ts

```diff
@@ -55,77 +52,92 @@ const useGetStartedMetadata = ({ checkDemoAppExists }: Props) => {
-      isComplete: settings?.signInExperienceCustomized,
-      onClick: () => {
-        navigate('/sign-in-experience');
+      {
+        id: 'customizeSignInExperience',
+        title: 'get_started.card3_title',
+        subtitle: 'get_started.card3_subtitle',
+        icon: isLightMode ? Customize : CustomizeDark,
+        buttonText: 'admin_console.general.customize',
+        isComplete: settings?.signInExperienceCustomized,
+        onClick: () => {
+          navigate('/sign-in-experience');
+        },
       },
-    },
-    {
-      id: 'configurePasswordless',
-      title: 'get_started.card4_title',
-      subtitle: 'get_started.card4_subtitle',
-      icon: isLightMode ? Passwordless : PasswordlessDark,
-      buttonText: 'admin_console.general.set_up',
-      isComplete: settings?.passwordlessConfigured,
-      onClick: () => {
-        navigate('/connectors');
+      {
+        id: 'configurePasswordless',
+        title: 'get_started.card4_title',
+        subtitle: 'get_started.card4_subtitle',
+        icon: isLightMode ? Passwordless : PasswordlessDark,
+        buttonText: 'admin_console.general.set_up',
+        isComplete: settings?.passwordlessConfigured,
+        onClick: () => {
+          navigate('/connectors');
+        },
       },
-    },
-    {
-      id: 'configureSocialSignIn',
-      title: 'get_started.card5_title',
-      subtitle: 'get_started.card5_subtitle',
-      icon: isLightMode ? OneClick : OneClickDark,
-      buttonText: 'admin_console.general.add',
-      isComplete: settings?.socialSignInConfigured,
-      onClick: () => {
-        navigate('/connectors/social');
+      {
+        id: 'configureSocialSignIn',
+        title: 'get_started.card5_title',
+        subtitle: 'get_started.card5_subtitle',
+        icon: isLightMode ? OneClick : OneClickDark,
+        buttonText: 'admin_console.general.add',
+        isComplete: settings?.socialSignInConfigured,
+        onClick: () => {
+          navigate('/connectors/social');
+        },
       },
-    },
-    {
-      id: 'checkFurtherReadings',
-      title: 'get_started.card6_title',
-      subtitle: 'get_started.card6_subtitle',
-      icon: isLightMode ? FurtherReadings : FurtherReadingsDark,
-      buttonText: 'admin_console.general.check_out',
-      isComplete: settings?.furtherReadingsChecked,
-      onClick: () => {
-        void updateSettings({ furtherReadingsChecked: true });
-        window.open('https://docs.logto.io/', '_blank');
+      {
+        id: 'checkFurtherReadings',
+        title: 'get_started.card6_title',
+        subtitle: 'get_started.card6_subtitle',
+        icon: isLightMode ? FurtherReadings : FurtherReadingsDark,
+        buttonText: 'admin_console.general.check_out',
+        isComplete: settings?.furtherReadingsChecked,
+        onClick: () => {
+          void updateSettings({ furtherReadingsChecked: true });
+          window.open('https://docs.logto.io/', '_blank');
```

Comment:
this should be the localized link

Reply:
yeah, let me fix it in another pr. Issue created
https://linear.app/silverhand/issue/LOG-3545/docs-link-in-get-started-should-be-localized